docs/spec/openapi.yml

```diff
@@ -584,6 +584,51 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  '/namespaces/{namespace}/database-engines/{name}/operator-version/preflight':
```

Comment:
Could the `preflight` be considered as `status`? I'm trying to find more obvious naming, bc when I look at this method it's hard to say what it does...  The method basically checks the current status of the upgrades and returns this information.

Reply:
> Could the preflight be considered as status?

No, not really 😅 .
The status of the upgrade is obtained from the DBEngine status. This API runs a set of preflight checks before you can actually perform the upgrade, that's why I went with `/operator-version/preflight` (as it returns the result of the preflight checks).

Another alternative I considered is `/check`, but that also sounds confusing - is it checking the status? Is it checking if it can upgrade? etc..

Reply:
For more context, this API gets the data that is displayed on this page: https://www.figma.com/file/g77gIX02jKysAzHzsDA9KW/Beta-Release?type=design&node-id=2686-27881&mode=design&t=vKuJZ56VnqZTgSAf-0

Reply:
Ok, I see. Let's stick with the `preflight` then